### ISSUE TITLE
Runtime: org.freedesktop.Sdk/21.08 is End Of Life. Update the version

### ISSUE THREAD

**Issue by MalayPalace** [flathub/com.sublimetext.three#36]
Repo is still using the EOL version as its runtime.

```
org.freedesktop.Platform 21.08 is no longer receiving fixes and security updates. Please update to a supported runtime version.
```

Can it be Updated to a version being supported **23.08** as of currently.

**Comment by janxkoci** [comment]
I second this, especially given ST4 is still not available on Flathub.

**Comment by ptomato** [comment]
Blocked on #37. Someone will have to figure that out first, or the upgrade is likely to break packages for all existing users.

**Comment by AsciiWolf** [comment]
I think that it would be a better idea to just remove Package Control at this point. Still using the ancient 21.08 runtime is unacceptable.

**Comment by ptomato** [comment]
Hi @AsciiWolf, please take another look at #37 — that will break packages for all existing users, which is also unacceptable. If you would like to help, it would be useful to figure out how to pop up a message after the upgrade telling users they need to reinstall Package Control themselves.

**Comment by janxkoci** [comment]
> all existing users

Assuming there is more than the three of us still using this...

Wasn't there an effort to bring ST4 to flathub? I remember it needed different name than this package, but I don't recall if there was other blocking issue..

**Comment by MalayPalace** [comment]
@janxkoci am not very experienced with flatpak but let me have a look at it. If you have any lead or anyone already working on it, lets collaborate.

**Comment by janxkoci** [comment]
I think there is no point in fixing this version, since it's not even supported upstream (by SublimeText devs). A much more fruitful effort will be to bring ST4 to flathub, which is disscussed in #26. The issue there seems very minor (picking the right name), and it can be done without including Package Control from the start, avoiding this issue here entirely.

**Comment by ptomato** [comment]
There is also an issue with showing a disclaimer on first run, see https://github.com/flathub/flathub/pull/2855

**Comment by janxkoci** [comment]
You are right, I noticed that later too. There is a link to something VScode does, which looked pretty simple (one if check, and show a file on first run). It should be easy to implement, I just don't know anything about flatpak packaging.

Moreover, I've noticed the argument from flathub about the disclaimer may also no longer be valid, as they clearly allow more prominent disclaimers now, like for example here for the Zed editor: https://flathub.org/apps/dev.zed.Zed
